docs/main/05-reference/06-statestore/01-statestore.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Documentation for the Kratix StateStore Custom Resources
+title: StateStores
+sidebar_label: StateStores
+id: intro
+---
+# StateStores
+
+The Kratix `StateStore` resources represent the different backing storage options
+for Kratix to write to. Kratix supports both [GitStateStore](./02-gitstatestore.md) for writing to Git repositories,
+and [BucketStateStores](./03-bucketstatestore.md) for writing to S3-compatible buckets. When registering a
+[Cluster](../02-clusters/01-clusters.md) you specify which `StateStore` it should
+write to. Checkout the docs for the supported StateStores:
+
+- [GitStateStore](./02-gitstatestore.md)
```

Comment:
you could link as `./gitstatestore` instead of `02-gitstatestore.md`; that allows for renaming the file without having to rewrite everywhere the link is used.

NOTE: it does not work with partials :'(

Reply:
@kirederik how does that work 🤔 is it using the ID from the page? does that mean even if the page moves to a different directory it works?